python/src/Rayleigh_doc.i.in

```diff
@@ -1,6 +1,9 @@
 %feature("docstring") OT::Rayleigh
 "Rayleigh distribution.
 
+Available constructors:
+    Rayleigh(*beta=1.0, gamma=0.0*)
```

Comment:
this is unnecessary when there is only one ctor

Reply:
it enables to mention the default values very easily...